### PR TITLE
chore(deps): upgrade @nestjs v7

### PR DESCRIPTION
**Pull request by 9renpoto**
- Added github actions
- upgrade @nestjs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,9 +11,9 @@
     "prepare": "tsc -p tsconfig.json"
   },
   "dependencies": {
-    "@nestjs/common": "^6.11.0",
-    "@nestjs/core": "^6.11.0",
-    "@nestjs/graphql": "6.5.3",
+    "@nestjs/common": "^7.0.5",
+    "@nestjs/core": "^7.0.5",
+    "@nestjs/graphql": "^7.0.14",
```
Comment (9renpoto) [inline package.json]: I want to move from dependency to devDdependency if u like :)

**Comment by 9renpoto** [comment]
@krislefeber Hi, please review if you like :)

**Comment by krislefeber** [comment]
Merging to v7 branch. I will be testing and releasing a new version shortly.

**Comment by krislefeber** [comment]
@9renpoto, thank you for the PR! This has been merged. I have synced up to now have the project follow the nestjs version, so it will be released under version 7.0.0

**Pull request merged**